src/index.js

```diff
@@ -0,0 +1,148 @@
+// Initial state of the page:
+
+const state = {
+  degrees: 35,
+  defaultMessage: "Pick Sky state and Temperature!",
+  city: "City of Seattle",
+}; 
+
+// Default values to display:
+
+document.getElementById("degrees").innerHTML = `${state.degrees}°`;
+document.getElementById("defaultMessage").innerHTML = `${state.defaultMessage}`;
+document.getElementById("city").innerHTML = `${state.city}`;
```

Comment:
Just a note, that these kinds of selections should go inside a [`https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event`](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event) event handler to ensure that the document is loaded prior to trying to select elements. 

In practice it rarely matters, but it could happen based on the order the browser loads things.